pkg/tools/assessmentopts.go

```diff
@@ -149,7 +156,7 @@ func (o *AssessmentOpts) GetCustomPoliciesDir(policyTypeName string, morePolicyT
 		log.Infof("{primary:%s} has no custom policies", o.Tool.Name())
 	} else {
 		// we need to unpack custom and lacework policies
-		err := extractArchives(dir, []string{"policies.tar.gz", "lacework_policies.tar.gz"})
+		err := extractArchives(dir, []string{"policies.zip", "lacework_policies.zip"})
```

Comment:
I am fine with this PR but one comment I would leave it always plan for backward compatibility when changing such things so we don't break stuff and releasing things to production doesn't require coordinated effort of releasing the CLI and API-Server. 

I agree we are not using custom policies anywhere but we are not sure how everyone is using, so better to follow some precautions